docs/code-generation/index.md

```diff
@@ -61,26 +69,27 @@ generate_file "/file.txt" {
 
 All code generation blocks use labels to identify the block and define where
 the generated code will be saved but they have different constraints depending
-on the [generation context](#generation-context).
+on the generation context.
+
+### 1. `Stack` Context Labels Constraints:
 
-For `stack` context, the labels must follow the constraints below:
+* The labels are in relative path and they must adhere to the format `<dir>/<filename>` or simply `<filename>`, that represents a relative path.
+* Regardless of the operating system, labels should be defined using the forward slash `/`.
+* Labels should not contain `../`, as code generation is confined to the stack and cannot navigate outside.
+* Labels should not start with `./`.
+* Labels should not represent symbolic links.
+* Each label must be unique within the stack hierarchy for blocks with `condition=true`.
+* It can’t be the stack.
```

Comment:
I'm not sure the new text reads better. The labels word is repeated in each sentence, even though we are enumerating the label constraints.

And this one does not read well:
`- It can’t be the stack.`

Which stack?
Maybe "It's not a stack" or "It can't be a stack" reads better?